recipes/eccodes-python/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "eccodes-python" %}
+{% set pypi_name = "eccodes-python" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: 587dd828ee5e13b6a7d35a362d6fd903d914dd0422f4010af95e8f45738e9143
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+
+test:
+  commands:
+    - test -f $(python -c "import site; print(site.getsitepackages()[0])")/eccodes/__init__.py  # [not win]
+
+about:
+  home: https://github.com/ecmwf/eccodes-python
+  license: Apache 2.0
+  license_family: Apache
+  summary: Python interface to read/write meteorological data in GRIB and BUFR.
```

Comment:
You need a license_file

Reply:
Thanks @beckermr - License file is added now and changed it to BSD which seems to be more popular on condo-forge.

Reply:
It needs to the be the license of the upstream code, whatever that is.

Reply:
Sorry. thought BSD was the license for condo-forge. Have updated it now to the Apache license of the upstream. License.txt is also updated

Reply:
Thanks, the recipes themselves are licenced as BSD-3-Clause but the files for this are added automatically ([example](https://github.com/conda-forge/root-feedstock/blob/master/LICENSE.txt)).

LGTM, thanks for the contribution! 🍰